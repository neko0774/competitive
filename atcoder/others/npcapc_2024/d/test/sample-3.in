12 10 8
1 3 2 6 3 6 7 7 5 5 4 7
12 4
7 10
4 2
9 8
9 9
8 3
4 9
10 2
