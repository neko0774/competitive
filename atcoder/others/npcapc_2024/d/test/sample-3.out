2741280
3007680
1503840
1916160
1972800
728640
1821600
621440
